### PR TITLE
Implement designs 8092 - adjust poster and spacing below nav

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
ADO card: ☑️ [AB#8092](https://dev.azure.com/S72/fefacba9-96b6-4af2-a53d-050ed453e13a/_workitems/edit/8092)
- [x] Has this been discussed with Delivery Team?

Designs: 
- 📱 [Mobile](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142ba7e15e60023340022d8)
- 📱 [Mobile with sponsor image](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142ba83fd3fd250f50e6216)
- 💊 [Tablet](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142baea05f63b262914e788)
- 💊 [Tablet with sponsor image](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142baeda13fc726053e405a)
- 🖥 [Desktop](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142bb2aceca8924c0b76f62)
- 🖥 [Desktop with sponsor image](https://app.zeplin.io/project/5fdaa165fbacf584a2ce4f8b/screen/6142bb2eb8582a2b4c2c93ee)

## Description of work
- Changes height between nav and top of film detail page at all breakpoints
- Changes film detail page poster size at all breakpoints
- Lowers breakpoint at which the sponsor logo moves to the right hand side of the film detail page

## Edge cases/Caveats/Known issues

- This work has been shown to design. Changes made from resulting feedback that arent in designs:
-  breakpoint at which the sponsor logo moves to the right hand side of the film detail page. This is was change from design to make this transition happen at the same time it goes from hamburger menu to desktop
- @stajs this PR includes the poster image on mobile. Bex is going to organise a meeting with Neville and a few others later and we'll discuss it then

## Checklist
- [x] CI tests are passing Github actions (inc. linting)
- [x] Key areas of the feature outlined for context and testing
- [x] If there are designs for this work are they noted here and in the ADO card 
- [x] Design review
- [x] Have checked this at multiple screen resolutions and range of browsers
- [ ] Moved ADO card to Dev/done, checked link to Github, tagged "Review"
- [x] Updated changelog (if applicable)
- [x] I promise to document any new feature toggles/configurations in the appropriate documentation

### CODE REVIEW

File: site/styles/_variables.scss
Comment (PeculiarGoat) [inline site/styles/_variables.scss:42]: @curtisadams28 does this have any impact on what you are currently working on?

@matthewthomsonnz have you tested what changing these values does to every page that uses them (film/bundle/tv season)?

Comment (curtisadams28) [reply site/styles/_variables.scss:42]: Should be fine

**Review by curtisadams28 — APPROVED** [review]
LGTM. Make sure to test on all the page types.

File: site/styles/_variables.scss
Comment (matthewthomsonnz) [reply site/styles/_variables.scss:42]: I had checked film/tv were consistent but not bundle.

The spacing between top of content and nav looks good at all breakpoints. Matches tv and film.

**Comment by matthewthomsonnz** [comment]
> LGTM. Make sure to test on all the page types.

Thanks Curtis. Tested but I'll wait for Jack's approval also.

**Comment by matthewthomsonnz** [comment]
> LGTM. Make sure to test on all the page types.

Thanks @curtisadams28 . Tested but I'll wait for @PeculiarGoat's approval also.

**Review by l0ud0gg — APPROVED** [review]
sweet with me. ran it up and gave it a quick look also 👍

**Comment by matthewthomsonnz** [comment]
@PeculiarGoat any objections? Will merge this arvo otherwise

**Review by PeculiarGoat — APPROVED** [review]
LGTM 👍

**Pull request merged**
